src/rgw/rgw_multi.cc

```diff
@@ -229,7 +229,7 @@ int abort_multipart_upload(RGWRados *store, CephContext *cct, RGWObjectCtx *obj_
   ret = store->send_chain_to_gc(chain, mp_obj.get_upload_id() , false);  // do it async
   if (ret < 0) {
     ldout(cct, 5) << "gc->send_chain() returned " << ret << dendl;
-    return ret;
+    return -EIO;
```

Comment:
Why convert this one to EIO?  It seems all we care about is ENOENT, so maybe check specifically for that?

Reply:
@dang The reason why I convert it to EIO is that I think the specific return code isn't significant for the caller except it is ENOENT, and we just need to return a error.

Reply:
It's never a good idea to hide error codes, IMO.  I'd prefer to see the codes passed through (except ENOENT filtered).

Reply:
@dang I think you are right, do you have any other concerns?